docs/ops/opset6.md

```diff
@@ -0,0 +1,159 @@
+# Operation Set `opset6` Specification {#openvino_docs_ops_opset6}
+
+This specification document describes `opset6` operation set supported in OpenVINO.
+Support for each particular operation from the list below depends on the capabilities available in a inference plugin
+and may vary among different hardware platforms and devices. Examples of operation instances are expressed as IR V10 xml
+snippets. Such IR is generated by the Model Optimizer. The semantics match corresponding nGraph operation classes
+declared in `namespace opset6`.
+
+
+## Table of Contents <a name="toc"></a>
+
+* [Abs](arithmetic/Abs_1.md)
+* [Acos](arithmetic/Acos_1.md)
+* [Acosh](arithmetic/Acosh_3.md)
+* [Add](arithmetic/Add_1.md)
+* [Asin](arithmetic/Asin_1.md)
+* [Asinh](arithmetic/Asinh_3.md)
+* [Assign](infrastructure/Assign_3.md)
```

Comment:
Assign-6

Reply:
I can't find the 6th version of the specification

~/repo/openvino/docs/ops$ find . | grep -i assign
./infrastructure/Assign_3.md